### PR TITLE
finished the lab

### PR DESCRIPTION
**Pull request by wosunkwo**

### CODE REVIEW

**Comment by wosunkwo** [comment]
- [x] Can successfully instantiate an empty linked list
- [x] Can properly insert into the linked list
- [x] The head property will properly point to the first node in the linked list
- [x] Can properly insert multiple nodes into the linked list
- [x] Will return true when finding a value within the linked list that exists
- [x] Will return false when searching for a value in the linked list that does not exist
- [x] Can properly return a collection of all the values that exist in the linked list

File: code401-challenges/src/main/java/code401/challenges/linkedlist/LinkedList.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
+package code401.challenges.linkedlist;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class LinkedList {
+    Node head;
```
Comment (evanslaton) [inline code401-challenges/src/main/java/code401/challenges/linkedlist/LinkedList.java:7]: You should get in the habit of declaring instance variable as either private or public
`public Node head;`
`public int size;`

File: code401-challenges/src/main/java/code401/challenges/linkedlist/LinkedList.java [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    int size;
+
+    LinkedList() {
+        //upon instantiation the head will point to null
+        this.head = null;
+        size = 0;
```
Comment (evanslaton) [inline code401-challenges/src/main/java/code401/challenges/linkedlist/LinkedList.java:13]: When assigning instance variables Java doesn't require you to use `this`. If you use `this` then you should use it everywhere. Consistency is something to strive for!

File: code401-challenges/src/main/java/code401/challenges/linkedlist/LinkedList.java [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    }
+
+    //function to insert a new node into the front of the linked list
+    public void insert(int value) {
+        try{
+            Node current = new Node();
+            current.value = value;
+            if (this.head == null) {
```
Comment (evanslaton) [inline code401-challenges/src/main/java/code401/challenges/linkedlist/LinkedList.java:21]: same comment as above, you use `this.head` once and then use `head` everywhere else. Try to remain consistent

File: code401-challenges/src/main/java/code401/challenges/linkedlist/Node.java [hunk 4]
```diff
@@ -0,0 +1,6 @@
+package code401.challenges.linkedlist;
+
+public class Node {
+    int value;
+    Node next;
+}
```
Comment (evanslaton) [inline code401-challenges/src/main/java/code401/challenges/linkedlist/Node.java:6]: This class doesn't have a constructor!

**Pull request merged**